docs/articles/commands/help_formatter.md

```diff
@@ -0,0 +1,80 @@
+---
+uid: commands_help_formatter
+title: Help Formatter
+---
+
+## Custom Help Formatter
```

Comment:
I believe help formatters also support DI, and if that's indeed the case, might be worth mentioning this.